lib/actions/reassemble.js

```diff
@@ -2,7 +2,33 @@
 const { messages } = require('elasticio-node');
 const ObjectStorageWrapperExtended = require('./utils-wrapper/ObjectStorageWrapperExtended');
 
-async function processAction(msg) {
+let timeHandle;
+let groupList = [];
+let delay;
+
+async function timer(this_) {
+  for (let i = 0; i < groupList.length; i += 1) {
+    const storage = new ObjectStorageWrapperExtended(this_);
+    // eslint-disable-next-line no-await-in-loop
+    const results = await storage.lookupParsedObjectById(groupList[i]);
+    const incomingData = {};
+    results.messages.forEach((message) => {
+      incomingData[message.messageId] = message.messageData;
+    });
+
+    // eslint-disable-next-line no-await-in-loop
+    await this_.emit('data', messages.newMessageWithBody({
+      groupSize: results.messages.length,
+      groupId: results.messages[0].groupId,
+      messageData: incomingData,
+    }));
+    // eslint-disable-next-line no-await-in-loop
+    await storage.deleteObjectById(groupList[i]);
+  }
+}
+
+async function processAction(msg, cfg) {
+  const { mode } = cfg;
```

Comment:
```suggestion
  const { mode = 'groupSize'} = cfg;
```
One last change: We should default to having the mode be set to group size in order to ensure backwards compatibility.